### PR TITLE
Add repo index pages

### PR DESCRIPTION
**Pull request by sharkwouter**
This adds an index page and a page for each package with info on them to the github pages where we host the repository like mentioned in https://github.com/pspdev/psp-packages/issues/155. I used the scripts there as an example to build this. This is some very basic html, filled using environment variables and envsubst. You can see the result here: https://sharkwouter.github.io/psp-packages/

I also added an empty style.css file. So if someone feels like making this look better, then can get started immediately after this has been merged.

### CODE REVIEW

**Comment by sharkwouter** [comment]
Don't mind the failed builds. I don't have a pspsdk docker container on my account, so it fails to pull that.

File: style.css
Comment (bucanero) [inline style.css:1]: Just a simple CSS to give some styling to the package table:

```css
h1, h2 {
     font-family: arial;
}

table {
     border-collapse: collapse;
     margin: 25px 0;
     font-size: 0.9em;
     font-family: sans-serif;
     min-width: 400px;
     box-shadow: 0 0 20px rgba(0, 0, 0, 0.15);
}

table thead tr {
     background-color: #009879;
     color: #ffffff;
     text-align: left;
}

table th, td {
     padding: 12px 15px;
}

table tbody tr {
     border-bottom: 1px solid #dddddd;
}

table tbody tr:nth-of-type(even) {
     background-color: #f3f3f3;
}

table tbody tr:last-of-type {
     border-bottom: 2px solid #009879;
}
```

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -65,17 +65,22 @@ jobs:
...
+        apk add --no-cache bash coreutils
         mkdir repo
         cp artifact-*/*.pkg.tar.gz repo/
+        ./make-repo-html.sh
         cd repo
         ${PSPDEV}/share/pacman/bin/repo-add pspdev.db.tar.gz *.pkg.tar.gz
         mv pspdev.db.tar.gz pspdev.db
         mv pspdev.files.tar.gz pspdev.files
-        tar -cvf ../repo.tar ./*
+        tar -cvf ../repo.tar ./*
```
Comment (bucanero) [inline .github/workflows/build.yml:83]:
```suggestion
        tar -cvf ../repo.tar ./*
```

File: style.css
Comment (sharkwouter) [reply style.css:1]: I've added some basic css based on this suggestion. Thanks!

**Review by bucanero — APPROVED** [review]

**Comment by tpimh** [comment]
Looks very nice! Hopefully, it will be useful.

**Pull request merged**